README.md

```diff
@@ -233,104 +233,12 @@ Example:
     ...
 </project>
 ```
-## Archetypes
+## Legacy archetypes
```

Comment:
Need to remove reference to archetypes at line 14. Also, I think we should not mention the archetypes here at all. Just have a new section "Getting started with Open Liberty" and point them to the starter. In the release notes, I will include information on the removal of the archetypes, and include a pointer to the last release documentation for reference.